src/GatewayConnectionManager.cpp

```diff
@@ -207,6 +207,15 @@ bool ConnectToLCG() {
 
   _lastConnectionAttempt = msNow;
 
+  if (Config::HasBackendLCGOverride()) {
+    std::string lcgOverride;
+    Config::GetBackendLCGOverride(lcgOverride);
+
+    ESP_LOGD(TAG, "Connecting to overridden LCG endpoint %.*s", lcgOverride.size(), lcgOverride.data());
+    s_wsClient->connect(lcgOverride.c_str());
+    return true;
```

Comment:
Semi-nitpick, but I guess this kinda also applies to the original `GatewayConnectionManager` behavior? 

If an invalid one is supplied, the log will assume a successful connection, even though absolutely no websocket server exists there. 🤔 

```[  3770][D][GatewayConnectionManager.cpp:276] Update(): [GatewayConnectionManager] Successfully verified auth token
[  3780][D][GatewayClient.cpp:19] GatewayClient(): [GatewayClient] Creating GatewayClient
[  3788][D][GatewayConnectionManager.cpp:214] ConnectToLCG(): [GatewayConnectionManager] Connecting to overridden LCG endpoint invalid.test.entry
...
[  3825][D][GatewayConnectionManager.cpp:286] Update(): [GatewayConnectionManager] Successfully connected to LCG
[  3875][E][WiFiGeneric.cpp:1582] hostByName(): DNS Failed for invalid.test.entry
[  3881][V][ssl_client.cpp:321] stop_ssl_socket(): Cleaning SSL connection.
[  3881][I][GatewayClient.cpp:149] _handleEvent(): [GatewayClient] Disconnected from API
[ 23792][D][GatewayConnectionManager.cpp:214] ConnectToLCG(): [GatewayConnectionManager] Connecting to overridden LCG endpoint invalid.test.entry
...
[ 23818][D][GatewayConnectionManager.cpp:286] Update(): [GatewayConnectionManager] Successfully connected to LCG
[ 23877][E][WiFiGeneric.cpp:1582] hostByName(): DNS Failed for invalid.test.entry
[ 23883][V][ssl_client.cpp:321] stop_ssl_socket(): Cleaning SSL connection.
[ 23883][I][GatewayClient.cpp:149] _handleEvent(): [GatewayClient] Disconnected from API```

Reply:
Outside the scope of this PR, but needs to be looked into, please make a seperate issue 😄